tests/conftest.py

```diff
@@ -281,9 +290,12 @@ def run_setup_file():
 
     def _run_setup_file(package_dir_path, *args):
         setup_file = str(package_dir_path / "setup.py")
-        return check_call(
-            (sys.executable, setup_file) + args, cwd=str(package_dir_path)
-        )  # nosec
+        with open(os.devnull, "w") as fp:
+            return subprocess.check_call(
+                (sys.executable, setup_file) + args,
+                cwd=str(package_dir_path),
+                stdout=fp,
+            )  # nosec
```

Comment:
```suggestion
            return subprocess.check_call(
                (sys.executable, setup_file) + args,
                cwd=str(package_dir_path),
                stdout=subprocess.DEVNULL,
            )  # nosec
```

Maybe [subprocess.DEVNULL](https://docs.python.org/3/library/subprocess.html#subprocess.DEVNULL)?

Reply:
Unfortunately, that is not available on Python 2 :cry:. If we start dropping it now, I'll change this code otherwise I'll fix it in PR #1243.

Reply:
Aw, right... I forgot about py2